lib/puppet/provider/mysql_grant/mysql.rb

```diff
@@ -77,6 +77,14 @@ def self.instances
                                     'REPLICATION_APPLIER', 'REPLICATION_SLAVE_ADMIN', 'RESOURCE_GROUP_ADMIN', 'RESOURCE_GROUP_USER', 'ROLE_ADMIN', 'SERVICE_CONNECTION_ADMIN',
                                     'SESSION_VARIABLES_ADMIN', 'SET_USER_ID', 'SHOW_ROUTINE', 'SYSTEM_USER', 'SYSTEM_VARIABLES_ADMIN', 'TABLE_ENCRYPTION_ADMIN', 'XA_RECOVER_ADMIN']
           sorted_privileges = ['ALL']
+
```

Comment:
May be better to have a comment here describing the issue that is is fixing, the same as you did with the fix just above it, otherwise it may confuse people in the future.

Reply:
How about instead of removing the comment you replace it with a link to the issue/issues?
That that it look's neater but still has an easy way to find out why the code's there if they want to know.

Reply:
I think a link would be kind of ugly. But I did replace it with a comment pointing at the github issue.